lib/app/features/feed/views/components/text_editor/components/custom_blocks/text_editor_poll_block/components/poll_answers_list_view/poll_answers_list_view.dart

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:ice/app/extensions/num.dart';
+import 'package:ice/app/features/feed/providers/poll/poll_answers_provider.dart';
+import 'package:ice/app/features/feed/views/components/text_editor/components/custom_blocks/text_editor_poll_block/components/poll_answer_item/poll_answer_item.dart';
```

Comment:
I would propose moving `poll_answer_item.dart` to this path:

`import 'package:ice/app/features/feed/views/components/text_editor/components/poll/poll_answer_item.dart';`